src/mumble/MainWindow.cpp

```diff
@@ -4152,6 +4165,10 @@ void MainWindow::versionCheck() {
 	new VersionCheck(false, this);
 }
 
+void MainWindow::togglePositionalAudio(const bool &newState) {
+	Global::get().s.bPositionalAudio = newState;
```

Comment:
This does not what the name suggests. If it is supposed to _toggle_ the positional audio state, this function shouldn't take a parameter for the new state as the new state is simply the inverted old state (e.g. `state = !state`). If in fact you want to be able to _set_ the state of positional audio, the function should be called `setPositionalAudio` or even better `enablePositionalAudio`.